tcms/telemetry/templates/telemetry/testing/breakdown.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block title %}{% trans "Testing Breakdown" %}{% endblock %}
+
+{% block contents %}
+
+    <div class="container-fluid container-cards-pf">
+
+        <form class="form-horizontal">
+            <div class="form-group">
+                <label for="id_select_product" class="col-md-1 col-lg-1">
+                    {% trans "Product" %}:
+                </label>
+                <div class="col-md-3">
+                    <select name="product" id="id_select_product" class="form-control selectpicker">
+                        <option value="">----------</option>
+                        {% for product in products %}
+                            <option value="{{ product.pk }}">
+                                {{ product.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+
+                <label for="id_select_test_plan" class="col-md-1 col-lg-1">
+                    {% trans "Test Plan" %}:
+                </label>
+                <div class="col-md-3">
+                    <select name="test_plan" id="id_select_test_plan" class="form-control selectpicker">
+                        <option value="">----------</option>
+                        {% for test_plan in test_plans %}
+                            <option value="{{ test_plan.pk }}">
+                                {{ test_plan.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+            </div>
+        </form>
+
+        <div class="col-md-3" style="padding: 15px; margin: 15px; text-align: center">
+            <div style="text-align: center; font-weight: bold"> {% trans "Automated/ Manual" %}</div>
+            <div id="total"></div>
+            <div> {% trans "Total" %}</div>
+
+            <div id="automated-bar" style="padding: 10px; font-weight: bold"></div>
+        </div>
+
+        <div class="col-md-4" style="padding: 15px; margin: 15px; text-align: center">
+            <div style="text-align: center; font-weight: bold">{% trans "Priorities" %}</div>
+
+            <div id="priorities-chart"></div>
```

Comment:
Both Priority and Category charts must include the Y axis with numbers.

When there are the same Priorities/Categories on the X axis (extreme case only 1) and I change the product the charts are updated but the scale of them is not. So I see 50 P1 cases for one product and 3 P1 cases for another product in the same scale.  Visually this makes it very hard to detect the change.

When Y axes are shown the numbers will change and cause a flicker on the screen.